gmail/src/views/login.ts

```diff
@@ -0,0 +1,95 @@
+import { notify, createKeyValueWidget } from "./helpers";
+import { State } from "../models/state";
+import { IMAGES_LOGIN } from "./icons";
+
+function repeat(str: string, n: number) {
+    let result = "";
+    while (n > 0) {
+        result += str;
+        n--;
+    }
+    return result;
+}
```

Comment:
What is that? :smile:

Reply:
Just to do a 
```python
"aa" * 5
```
Like we do in Python, but "string".repeat(5) does not work when executed by the addon :/

Reply:
Moved to utils